tests/system/lab/build_dirs

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+ARCHIVE='1.7.0RC/buster/armv7l'
+POST='-1.7.0-armv7l.deb'
+GUI_POST='-1.7.0.deb'
+#ARCHIVE='nightly/buster/armv7l'
+#POST='-1.6.0-armv7l.deb'
+#GUI_POST='-1.6.0next.deb'
+
+cd
+rm -rf foglamp-bak
+mv foglamp foglamp-bak
+mkdir -p foglamp/plugins/other
+cd foglamp/plugins/other
+wget -r --no-parent http://archives.dianomic.com/$ARCHIVE
```

Comment:
Instead of wget, Better would be:

1) sudo cp /etc/apt/sources.list /etc/apt/sources.list.bak
2) echo "deb http://archives.dianomic.com/nightly/buster/armv7l/ /" | sudo tee -a /etc/apt/sources.list
- replace nightly with arg
3) sudo apt update
4) and  run fresh_install with direct sudo apt install `{x}` instead of local